benchmarks/decoders/generate_readme_chart.py

```diff
@@ -42,9 +42,9 @@ def main() -> None:
     gop_sizes = [600]
     durations = [10]
     pix_fmts = ["yuv420p"]
-    ffmpeg_path = "/usr/local/bin/ffmpeg"
-    videos_path = "/tmp/videos"
-    shutil.rmtree(videos_path)
+    ffmpeg_path = "ffmpeg"
+    videos_path = "/tmp/torchcodec_benchmarking_videos"
```

Comment:
Maybe this should be renamed to videos_dir_path

I like path over files because it tells the user it's a full path and not the filename only

Reply:
Yeah, good point, I'll make one use `dir` and the other `file`.